core/src/banking_stage.rs

```diff
@@ -2062,6 +2068,31 @@ impl BankingStage {
         Ok(())
     }
 
+    fn try_receive_deserialized_packets(
+        deserialized_packet_receiver: &BankingPacketReceiver,
+        recv_timeout: Duration,
+        capacity: usize,
+    ) -> Result<Vec<ReceivePacketResults>, RecvTimeoutError> {
+        let stop = Instant::now() + recv_timeout;
+        let mut deserialized_packet_batches =
+            vec![deserialized_packet_receiver.recv_timeout(recv_timeout)?];
+
+        let mut received_capacity = deserialized_packet_batches[0].deserialized_packets.len();
```

Comment:
nit: received_capacity -> received_count, since it's tracking the current number of received packets, not a capacity